pages/404.tsx

```diff
@@ -0,0 +1,42 @@
+import Wrapper from "@components/SDCWrapper";
+import SvgErrorIcon from "@components/icons/ErrorIcon";
+import { Button, Box, Center, Heading, Text } from "@chakra-ui/react";
+
+type ComponentProps = {
+    session: Record<string, unknown>;
+};
+
+export default function Custom404(props: ComponentProps): JSX.Element {
+    const goToHome = () => {
+        window.location.href = "/";
+    };
+
+    return (
+        <Wrapper session={props.session}>
+            <Center minHeight="85vh" align="center">
+                <Box align="center" width="70%">
+                    <SvgErrorIcon />
+                    <Heading size="md" margin="3% 0% 3% 0%">
```

Comment:
little chakra-ism, `my` is the same as margin for y-axis. We also like to use unitless ticks (1, 2, 3, 4...), especially when we are just eyeballing.
```suggestion
                    <Heading size="md" my={3}>
```